### DISCUSSION TITLE
Part4. ch02. 테라폼 관련 질문입니다.

### DISCUSSION THREAD

**Discussion by dltkddbs87** [tedilabs/fastcampus-devops#73] [Q&A] [answered]
테라폼이 aws 시스템에서 커버(?)할 수 있는 범위가 어디까지인가요?
전체 서비스가 다 가능한지 일부만 가능하다면 어떤 시스템이 가능한지 궁금합니다.

**Comment by posquit0** [comment, marked as answer]
안녕하세요! [테라폼 AWS 프로바이더 공식 문서](https://registry.terraform.io/providers/hashicorp/aws/latest/docs)에서 확인하실 수 있습니다.
프로바이더도 계속해서 버전 업이 되면서 AWS 신규 기능과 리소스를 따라잡고 있습니다. :)
